File: mis_conexiones/javascript/mysql.js
const mysql = require('mysql');

const connection = mysql.createConnection({
  host: 'localhost',
  user: 'root',
  password: '12345',
  database: 'tienda'
});

connection.connect();

connection.query('SELECT * FROM tabla', (error, results) => {
  if (error) throw error;
  console.log(results);
});

connection.end();

File: mis_conexiones/python/mysql.py
import mysql.connector

conn = mysql.connector.connect(
    host='localhost',
    user='root',
    password='12345',
    database='tienda'
)

cursor = conn.cursor()
cursor.execute('SELECT * FROM tabla')

for row in cursor.fetchall():
    print(row)

conn.close()
